pkg/controller/disruption/disruption_test.go

```diff
@@ -1538,12 +1538,10 @@ func TestStalePodDisruption(t *testing.T) {
 	}
 	for name, tc := range cases {
 		t.Run(name, func(t *testing.T) {
-			_, ctx := ktesting.NewTestContext(t)
-			ctx, cancel := context.WithCancel(ctx)
-			defer cancel()
-			dc, _ := newFakeDisruptionControllerWithTime(ctx, now)
-			go dc.Run(ctx)
-			if _, err := dc.coreClient.CoreV1().Pods(tc.pod.Namespace).Create(ctx, tc.pod, metav1.CreateOptions{}); err != nil {
+			tCtx := ktesting.Init(t)
+			dc, _ := newFakeDisruptionControllerWithTime(tCtx, now)
+			go dc.Run(tCtx)
+			if _, err := dc.coreClient.CoreV1().Pods(tc.pod.Namespace).Create(tCtx, tc.pod, metav1.CreateOptions{}); err != nil {
```

Comment:
I think calling the context variable `tCtx` or (in other places) `kCtx` is useful because it makes it clear that this is not an ordinary `context.Context`.

The downside is the amount of code churn, like in this place: this line wouldn't change at all if the variable was still called `ctx`.

@aojea: any preference from you here?

Reply:
Forget about `kCtx`, I dropped the idea of having two different context types. Only `tCtx TContext` remains, but the question how to name the variable and parameter remains.